src/gui/instrument/InstrumentTrackWindow.cpp

```diff
@@ -289,7 +289,7 @@ InstrumentTrackWindow::InstrumentTrackWindow( InstrumentTrackView * _itv ) :
 		flags |= Qt::MSWindowsFixedSizeDialogHint;
 		// any better way than this?
 	} else {
-		subWin->setMaximumSize(m_instrumentView->maximumHeight() + 12, m_instrumentView->maximumWidth() + 208);
+		subWin->setMaximumSize(m_instrumentView->maximumWidth() + 12, m_instrumentView->maximumHeight() + 208);
```

Comment:
Would it be better to use `setFixedSize` here rather than calling both `setMinimumSize` and `setMaximumSize`?

Reply:
I believe this has got something to do with resize ability